tests/testthat/test-utils-formula.R

```diff
@@ -26,6 +26,18 @@ test_that("h_extract_covariance_terms returns a list of possible covariance term
     h_extract_covariance_terms(a ~ a + us(a | g / s)),
     list(us = quote(us(a | g / s)))
   )
+
```

Comment:
maybe does not hurt to add an integration test for the concrete user bug in the relevant test file?